data/strings/strings.txt

```diff
@@ -26894,3 +26894,390 @@
+    et = Kas teate, et saate selle valimiseks pikalt puudutada mis tahes kohta kaardil?
+    eu = Ba al dakizu mapako edozein toki luze sakatu dezakezula hura hautatzeko?
+    fa = آیا می‌دانید که می‌توانید روی هر مکانی روی نقشه ضربه طولانی بزنید تا آن را انتخاب کنید؟
+    fi = Tiesitkö, että voit valita mitä tahansa paikkaa kartalla pitkään napauttamalla sen?
+    fr = Savez-vous que vous pouvez appuyer longuement sur n'importe quel endroit de la carte pour le sélectionner ?
+    he = האם אתה יודע שאתה יכול ללחוץ ארוכות על כל מקום במפה כדי לבחור אותו?
+    hi = क्या आप जानते हैं कि आप मानचित्र पर किसी भी स्थान को चुनने के लिए उस पर देर तक टैप कर सकते हैं?
+    hu = Tudja, hogy a térkép bármely pontjára hosszan koppintva kiválaszthatja azt?
+    id = Tahukah Anda bahwa Anda dapat mengetuk lama tempat mana pun di peta untuk memilihnya?
+    it = Sai che puoi toccare a lungo qualsiasi luogo sulla mappa per selezionarlo?
+    ja = 地図上の任意の場所を長押しして選択できることをご存知ですか?
+    ko = 지도의 아무 곳이나 길게 탭하여 선택할 수 있다는 것을 알고 계셨나요?
+    mr = तुम्हाला माहीत आहे का की तुम्ही नकाशावरील कोणतेही ठिकाण निवडण्यासाठी लांब-टॅप करू शकता?
+    nb = Vet du at du kan trykke lenge på et hvilket som helst sted på kartet for å velge det?
+    nl = Wist u dat u op elke plaats op de kaart lang kunt tikken om deze te selecteren?
+    pl = Czy wiesz, że możesz długo dotknąć dowolnego miejsca na mapie, aby je wybrać?
+    pt = Sabia que pode tocar longamente em qualquer lugar do mapa para selecioná-lo?
+    pt-BR = Você sabia que pode tocar longamente em qualquer lugar do mapa para selecioná-lo?
+    ro = Știți că puteți atinge lung orice loc de pe hartă pentru a-l selecta?
+    ru = Знаете ли вы, что любое место на карте можно выбрать, подержав там палец в течение секунды?
+    sk = Viete, že na ľubovoľné miesto na mape ho môžete vybrať dlhým klepnutím?
+    sv = Vet du att du kan trycka länge på vilken plats som helst på kartan för att välja den?
+    sw = Je, unajua kwamba unaweza kugonga kwa muda mrefu sehemu yoyote kwenye ramani ili kuichagua?
+    th = คุณรู้ไหมว่าคุณสามารถแตะสถานที่ใดก็ได้บนแผนที่ค้างไว้เพื่อเลือกได้
+    tr = Haritadaki herhangi bir yeri seçmek için uzun süre dokunabileceğinizi biliyor musunuz?
+    uk = Чи знаєте ви, що ви можете довго торкатися будь-якого місця на карті, щоб вибрати його?
+    vi = Bạn có biết rằng bạn có thể nhấn và giữ bất kỳ địa điểm nào trên bản đồ để chọn địa điểm đó không?
+    zh-Hans = 您知道您可以长按地图上的任何位置来选择它吗？
+    zh-Hant = 您知道您可以長按地圖上的任何位置來選擇它嗎？
+
+  [app_tip_05]
+    comment = App tip #05
+    tags = android
+    en = Do you know that you can select your current location on the map?
+    ar = هل تعلم أنه يمكنك تحديد موقعك الحالي على الخريطة؟
+    be = Ці ведаеце вы, што вы можаце выбраць сваё бягучае месцазнаходжанне на карце?
+    bg = Знаете ли, че можете да изберете текущото си местоположение на картата?
+    ca = Saps que pots seleccionar la teva ubicació actual al mapa?
```

Comment:
Sabíeu que podeu seleccionar la vostra ubicació actual al mapa?

Reply:
Thanks @fitojb! Please confirm also the Catalonian translation of Tip 4? (`Saps que pots tocar llargament qualsevol lloc del mapa per seleccionar-lo?`)

Can we change it to: `Sabíeu que podeu tocar llargament qualsevol lloc del mapa per seleccionar-lo?` ??

Reply:
Translation modified !!

Reply:
> Can we change it to: `Sabíeu que podeu tocar llargament qualsevol lloc del mapa per seleccionar-lo?` ??

Yes; in general, per Softcatalà's guidelines we use the vós treatment when referring to users. Spanish should use usted rather than tú. :)

Reply:
Thanks @fitojb! In Spanish we also use the formal "Usted" form... Have you detected any string in Spanish where this formal form is not used?